docs/usage/security/abstract-authentication-middleware.rst

```diff
@@ -1,83 +1,91 @@
-AbstractAuthenticationMiddleware
-================================
+==================================
+Implementing Custom Authentication
+==================================
 
-Litestar exports :class:`AbstractAuthenticationMiddleware <.middleware.authentication.AbstractAuthenticationMiddleware>`,
-which is an Abstract Base Class (ABC) that implements the :class:`MiddlewareProtocol <.middleware.base.MiddlewareProtocol>`.
+Litestar exports :class:`~.middleware.authentication.AbstractAuthenticationMiddleware`, which is an
+:term:`abstract base class` (ABC) that implements the :class:`~.middleware.base.MiddlewareProtocol`.
 To add authentication to your app using this class as a basis, subclass it and implement the abstract method
-:meth:`authenticate_request <.middleware.authentication.AbstractAuthenticationMiddleware.authenticate_request>`:
+:meth:`~.middleware.authentication.AbstractAuthenticationMiddleware.authenticate_request`:
 
-.. code-block:: python
+.. dropdown:: Click to see an example
```

Comment:
Not a fan of these dropdowns everywhere tbh :eyes: 
When I'm reading docs, I *want* to see the code. This just makes it harder to skim and see what's actually going on. I'd be pretty annoyed if I'd have to expand every code example like this. Also, since the text below this is referring to the content of the dropdown, having it hidden by default feels a bit odd. 

> As you can see [...]

But I *can't* see :grimacing: 

IMO dropdowns should be used if there's some additional information, not on the main content.